pkg/storaged/dialog.jsx

```diff
@@ -728,9 +728,9 @@ export const SelectSpaces = (tag, title, options) => {
 
 export const SelectSpace = (tag, title, options) => {
     return {
-        tag: tag,
-        title: title,
-        options: options,
+        tag,
+        title,
+        options,
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)